lib/puppet/indirector/json.rb

```diff
@@ -52,7 +52,7 @@ def load_json_from_file(file, key)
     json = nil
 
     begin
-      json = File.read(file)
+      json = File.read(file).force_encoding(Encoding::UTF_8)
```

Comment:
I take it back, it is there https://github.com/puppetlabs/puppet/blob/8bb8c227ff46b6018376a96531921ede65f8a942/lib/puppet/file_system/windows.rb#L96-L102 

However, we may want to consider moving this logic into read.